src/interface.cr

```diff
@@ -7,7 +7,6 @@ module PlaceCalendar
       [] of User
     end
 
-    # do we need this
     def get_user(id : String?, **options) : User?
     end
```

Comment:
This will give a compile-time error if this method is used. 

```suggestion
    def get_user(id : String?, **options) : User?
      {{ raise "Unimplemented" }}
    end
```

Reply:
@caspiano the `PlaceCalendar::Interface` class is never actually instantiated, the `PlaceCalendar::Google`, and `PlaceCalendar::Office365` classes just inherit from it. I’m guessing it’d be smarter to make it an abstract class right? Perhaps even making all those methods abstract methods too? 🤔

Reply:
nice one! @tob1k